### PR TITLE
Fix Stripe "email" error

### PR DESCRIPTION
**Pull request by franjohn21**
Fixes https://github.com/SocialiteProviders/Providers/issues/681 - missing "email" in account API response.

### CODE REVIEW

File: src/Stripe/Provider.php [hunk 1]
```diff
@@ -64,10 +64,15 @@ protected function mapUserToObject(array $user)
         } elseif (isset($user['display_name'])) { // original location
             $nickname = $user['display_name'];
         }
+
```
Comment (franjohn21) [inline src/Stripe/Provider.php:67]:
```suggestion
```

File: src/Stripe/Provider.php [hunk 2]
```diff
@@ -64,10 +64,14 @@ protected function mapUserToObject(array $user)
...
             $nickname = $user['display_name'];
         }
+        $email = null;
+        if (isset($user['email'])) {
+            $email = $user['email'];
+        }
 
         return (new User())->setRaw($user)->map([
             'id'   => $user['id'], 'nickname' => $nickname,
-            'name' => null, 'email' => $user['email'], 'avatar' => null,
```
Comment (atymic) [inline src/Stripe/Provider.php:70]: Could you split these props onto one per line.

File: src/Stripe/Provider.php [hunk 3]
```diff
@@ -64,10 +64,14 @@ protected function mapUserToObject(array $user)
...
+            'name' => null, 'email' => $email, 'avatar' => null,
```
Comment (atymic) [inline src/Stripe/Provider.php:74]:
```suggestion
            'name' => null, 'email' => $user['email'] ?? null, 'avatar' => null,
```

**Review by atymic — CHANGES_REQUESTED** [review]

**Comment by atymic** [comment]
Looks good just use null coalesce instead of the conditional.

**Review by atymic — COMMENTED** [review]
Could you also update the docs to note the email field is optional please?

**Review by atymic — APPROVED** [review]

**Pull request merged**